sw/airborne/firmwares/rotorcraft/guidance/guidance_indi.c

```diff
@@ -57,6 +62,10 @@ float roll_filtdd = 0;
 float pitch_filt = 0;
 float pitch_filtd = 0;
 float pitch_filtdd = 0;
+float T_act = 0;
```

Comment:
minor note: per our conventions, variables are usually all lower case...
In any case `thrust_act` might be more readable... on capital T I first thought about time period or Tesla